.github/workflows/ruff.yml

```diff
@@ -0,0 +1,11 @@
+name: Ruff Linter
+on: [push, pull_request]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: astral-sh/ruff-action@v3
+        with:
+          version: 0.11.4
+          args: "check --output-format=github"
```

Comment:
Does ``--output-format=github`` mean that any needed format changes will be displayed in the GitHub Actions tab, e.g. https://github.com/NVIDIA/NeMo-Curator/actions/runs/14275900173/job/40018701685?pr=628 ?